week1_assignment/index.html

```diff
@@ -0,0 +1,252 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>시연의 블로그</title>
+    <script src="https://kit.fontawesome.com/102eb053c5.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css"/>
+</head>
+<body>
+    <!--헤더-->
+    <header>
+        <img id="hamster" src="https://www.pngarts.com/files/5/Hamster-Download-PNG-Image.png" alt="블로그 로고">
+        <h1>시연의 블로그</h1>
+        <button type="button" class="hamburger">
+            <i class="fa-solid fa-bars"></i>
+        </button>
+    </header>
+    <main>
+        <!--네비게이션-->
+        <nav>
+            <h2>분류</h2>
+            <div>
+                <a href="#블로그시작" class="nav-btn1">블로그</a><br>
+                <a href="#고양이" class="nav-btn2">고양이</a><br>
+                <a href="#아이유" class="nav-btn3">아이유</a>
+                
+            </div>
+            
+        </nav>
+        <!--블로그 시작 영역-->
+        <section id="블로그시작" class="first_box">
+            <h3>유애나의 블로그입니다^^</h3>
+                <div class="first_container">
+                        <img src="https://herb-api.hankookilbo.com/api/attaches/image/group/b0a0d85f-dc61-4e39-b356-ea7389ab71d0" alt="아이유 사진1" class="item1">
+                        <img src="https://topclass.chosun.com/news/photo/202305/31799_43125_2238.jpg" alt="아이유 사진2" class="item2">
+                        <img src="https://cdn.straightnews.co.kr/news/photo/202402/244929_148103_52.jpg" alt="아이유 사진3" class="item3">
+                        <img src="https://img1.yna.co.kr/photo/yna/YH/2021/12/30/PYH2021123006320000500_P4.jpg" alt="아이유 사진4" class="item4">
+                        <img src="https://cdn.newsquest.co.kr/news/photo/202105/84576_63389_4212.jpg" alt="아이유 사진5" class="item5">
+                </div>
+        </section>
+        <!--전체 영역-->
+        <section id="전체" class="second_box">
+            <h3>전체</h3>
+            <!--고양이 사진들-->
+                <div class="second_container">
+                    <div class="img_block">
+                        <img src="https://img.animalplanet.co.kr/news/2019/11/28/700/f9in35p5660ce423x290.jpg" alt="피곤한 고양이">
+                        <div class="text">
+                            고양이<br>
+                            좋아요 10개<br>
+                            <img id="heart" src="./heart-solid.svg" alt="좋아요 아이콘">
+                        </div>
```

Comment:
```suggestion
<div class="text">
  <p>고양이</p>
  <p>좋아요 10개</p>
  <img id="heart" src="./heart-solid.svg" alt="좋아요 아이콘">
</div>
```

`<p>` 태그를 사용하는게 단락을 의미해 의미론적으로 적절해요.
스타일링도 `<br>`보다는 margin을 통해서 간편하게 간격 조절이 가능합니다! `<br>`태그는 단순 줄바꿈만 제공해 구조적 의미가 없어 유지보수할 때 불리합니다!ㅎ